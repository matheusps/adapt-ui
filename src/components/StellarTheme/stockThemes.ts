export const light = {
  color: {
    base: {
      default: '#fafafa',
      darken: '#cecece',
      lighten: '#ffffff',
      contrast: '#000000',
    },
    primary: {
      default: '#256EFF',
      darken: '#113274',
      lighten: '#AFCAFF',
      contrast: '',
    },
  },
}
