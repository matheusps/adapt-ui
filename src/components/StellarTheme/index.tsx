import React, { FunctionComponent } from 'react'
import { ThemeProvider } from 'styled-components'

import { light } from './stockThemes'

/**
 * Wraps around Theme-provider from styled-components 💅🏻
 */
const StellarTheme: FunctionComponent<StellarThemeProps> = ({ children }) => {
  return (
    <ThemeProvider theme={light}>
      <>{children}</>
    </ThemeProvider>
  )
}

export default StellarTheme
