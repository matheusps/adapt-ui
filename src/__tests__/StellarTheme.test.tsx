import React from 'react'
import { render } from 'react-testing-library'
import StellarTheme from '../components/StellarTheme'

describe('Container component', () => {
  const component = render(
    <StellarTheme>
      <p>Test</p>
    </StellarTheme>
  )

  it('should render', () => {
    expect(component).toBeTruthy()
  })

  it('should match snapshot', () => {
    expect(component.asFragment()).toMatchSnapshot()
  })
})
