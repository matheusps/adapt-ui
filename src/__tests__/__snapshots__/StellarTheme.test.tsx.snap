// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container component should match snapshot 1`] = `
<DocumentFragment>
  <p>
    Test
  </p>
</DocumentFragment>
`;
